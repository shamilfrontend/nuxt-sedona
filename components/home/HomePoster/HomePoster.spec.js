import { mount } from '@vue/test-utils'
import Component from './index'

describe('HomePoster', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Component)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
