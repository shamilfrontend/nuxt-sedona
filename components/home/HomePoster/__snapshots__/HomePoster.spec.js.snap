// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePoster should match snapshot 1`] = `<div class="home-poster"><img src="./img/title-intro.png" alt="Welcome Sedona" class="home-poster__image"></div>`;
